### PR TITLE
Install OSQuery in our AMIs

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
https://github.com/grapl-security/issue-tracker/issues/604

### What changes does this PR make to Grapl? Why?
Add `osquery` and a dummy-config-ready-to-be-edited to our AMIs. 

### How were these changes tested?
wishful thinking

### CODE REVIEW

File: packer/nomad-server-client/image.pkr.hcl [hunk 1]
```diff
@@ -80,6 +56,40 @@ variable "aws_build_region" {
   default     = "us-east-1"
 }
 
+####################
+# Program Versions
```
Comment (wimax-grapl) [inline packer/nomad-server-client/image.pkr.hcl:60]: i couldn't figure out what to call this. Dependency versions?

Comment (christophermaier) [reply packer/nomad-server-client/image.pkr.hcl:60]: "Software Versions"

File: packer/nomad-server-client/setup_osquery.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+# Add repo
+curl -L https://pkg.osquery.io/rpm/GPG | sudo tee /etc/pki/rpm-gpg/RPM-GPG-KEY-osquery
+sudo yum-config-manager --add-repo https://pkg.osquery.io/rpm/osquery-s3-rpm.repo
+sudo yum-config-manager --enable osquery-s3-rpm-repo
+
+sudo yum install --assumeyes osquery
+
+# Ensure it's all working correctly
+osqueryi --version
```
Comment (christophermaier) [inline packer/nomad-server-client/setup_osquery.sh:16]: Is this the actual name of the binary, or a typo?

**Review by christophermaier — APPROVED** [review]
Seems fine for what it is, but would be more useful with a real configuration in place.

File: packer/nomad-server-client/setup_osquery.sh (hunk 2 above)
Comment (wimax-grapl) [reply packer/nomad-server-client/setup_osquery.sh:16]: Good question, but yes, it is correct!

**Pull request merged**